test/common/request_info/BUILD

```diff
@@ -21,6 +22,7 @@ envoy_cc_test(
     name = "request_info_impl_test",
     srcs = ["request_info_impl_test.cc"],
     deps = [
+        ":test_int_accessor_interface",
```

Comment:
s/interface/lib/

Reply:
Done.  

Having said that, would you be willing to give me a sense of when envoy wants targets named _interface vs. _lib?  I assumed it was if the target was header only, but it sounds like I got that wrong.

Reply:
yep, interface is reserved for virtual classes (the vast majority, if not all under this tree https://github.com/envoyproxy/envoy/tree/master/include/envoy, lib are for concrete implementations, even if just in a header file.